docs/pages/upcoming-releases.mdx

```diff
@@ -130,235 +54,160 @@ Imports from the new names will be allowed shortly, and support for the old name
-      <td>-</td>
-      <td>Either define disabled in your `Option` (in `items`), or use `disabledKeys`</td>
+      <td colSpan={4}>Calendar props</td>
     </tr>
     <tr>
-      <td>`EmptyState`</td>
-      <td>`layoutContext`</td>
+      <td>`CalendarRange`</td>
+      <td>
+        `fromDate`, `fromMonth`, `fromYear`, `toDate`, `toMonth`, `toYear`, `initialFocus`,
+        `onDayKeyUp`, `onDayKeyPress`, `onDayPointerEnter`, `onDayPointerLeave`, `onDayTouchCancel`,
+        `onDayTouchEnd`, `onDayTouchMove`, `onDayTouchStart`
+      </td>
       <td>-</td>
-      <td>This prop no longer has any effect</td>
+      <td rowSpan={2}>React Day Picker has had these props deprecated</td>
     </tr>
     <tr>
-      <td>`FilterBarContext`</td>
-      <td>`toggleOpenFilter`</td>
+      <td>`CalendarSingle`</td>
+      <td>
+        `fromMonth`, `fromYear`, `toDate`, `toMonth`, `toYear`, `initialFocus`, `onDayKeyUp`,
+        `onDayKeyPress`, `onDayPointerEnter`, `onDayPointerLeave`, `onDayTouchCancel`,
+        `onDayTouchEnd`, `onDayTouchMove`, `onDayTouchStart`
+      </td>
       <td>-</td>
-      <td>Use `setFilterOpenState` instead</td>
     </tr>
   </tbody>
 </table>
 
-### Other changes
-
-Support for old import paths will be dropped. This includes imports from `@kaizen/components/future`, or any versioned entry point (e.g. `@kaizen/components/v3/actions`)
-
-A codemod will be provided to make the migration off these.
-
-## `@kaizen/design-tokens@v11`
-
-September 1, 2025
-
-### Utils being removed
+### Deleted utils and exports
 
 <table>
   <thead>
     <tr>
-      <th>Util/Component</th>
+      <th>Util/Export</th>
       <th>Reason</th>
     </tr>
   </thead>
   <tbody>
     <tr>
-      <td>`ThemeManager`</td>
-      <td>
-        Not needed if you are using `KaizenProvider` from `@kaizen/components` or `defaultPreset`
-        from next-services.
-      </td>
-    </tr>
-    <tr>
-      <td>`ThemeProvider`</td>
-      <td>
-        Not needed if you are using `KaizenProvider` from `@kaizen/components` or `defaultPreset`
-        from next-services.
-      </td>
-    </tr>
-    <tr>
-      <td>`Theme`</td>
-      <td>
-        <ul>
-          <li>
-            Not needed if you are using `KaizenProvider` from `@kaizen/components` or
-            `defaultPreset` from next-services.
-          </li>
-          <li>If necessary, import from `@kaizen/design-tokens/js`</li>
-        </ul>
-      </td>
+      <td>`warn` util in `Modal/util/console.ts`</td>
+      <td>No longer needed</td>
     </tr>
     <tr>
-      <td>`heartTheme`</td>
-      <td>
-        <ul>
-          <li>
-            Not needed if you are using `KaizenProvider` from `@kaizen/components` or
-            `defaultPreset` from next-services.
-          </li>
-          <li>If necessary, import from `@kaizen/design-tokens/js`</li>
-        </ul>
-      </td>
+      <td>`useResizeObserver`</td>
+      <td>Use native `ResizeObserver` DOM API instead</td>
```

Comment:
Perhaps a link to MDN docs here? https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver